### PR TITLE
Added headings in the client/lxd_instances.go file that were missing

### PR DESCRIPTION
**Pull request by ibilalkayy**
Hey there,

I just added headings in the **client/lxd_instances.go** file that were missing. With these headings, it will be easy to understand what these functions do.

Thank you!

### CODE REVIEW

**Comment by tomponline** [comment]
Hi @ibilalkayy I wonder is it possible for you to submit multiple commits for these changes inside a single PR rather than opening lots of PR. This way we can review and test the changes in one shot. Thanks

**Comment by ibilalkayy** [comment]
@tomponline  Alright. I will send one PR. Thank you, for letting me know!

File: client/lxd_instances.go [hunk 1]
```diff
@@ -609,6 +612,8 @@ func (r *ProtocolLXD) CreateInstance(instance api.InstancesPost) (Operation, err
 	return op, nil
 }
 
+// tryCreateInstance attempts to create a new instance on multiple target servers specified by their URLs.
```
Comment (tomponline) [inline client/lxd_instances.go:615]: Please can you split the last 2 comments into their own commit to aid backporting.

**Review by tomponline — CHANGES_REQUESTED** [review]
We need to split these changes up so that last 2 comments can be backported.

**Comment by ibilalkayy** [comment]
In **lxd_instances.go** file, I added 4 headings above the 4 functions. Should I split these comments into 4 commits?

**Comment by ibilalkayy** [comment]
I splitted 4 comments into 4 commits.

**Review by tomponline — APPROVED** [review]
Thanks!

**Pull request merged**